src/test/regress/expected/failure_multi_row_insert.out

```diff
@@ -43,10 +43,10 @@ SELECT citus.mitmproxy('conn.onQuery(query="^INSERT").kill()');
 (1 row)
 
 INSERT INTO distributed_table VALUES (1,1), (1,2), (1,3);
-ERROR:  server closed the connection unexpectedly
+ERROR:  connection error: localhost:9060
```

Comment:
now that failure PRs merged, I pushed 1 more commit that updates the error messages here.